ui/page/start_page.go

```diff
@@ -329,38 +409,187 @@ func (sp *startPage) layoutMobile(gtx C) D {
+		layout.Rigid(func(gtx C) D {
+			if sp.currentPage == 3 {
+				return layout.Inset{Bottom: values.MarginPaddingMinus145, Left: values.MarginPadding20, Top: values.MarginPadding20}.Layout(gtx, sp.pageHeaderLayout)
+			}
+			return D{}
+		}),
+		layout.Rigid(func(gtx C) D {
+			return sp.pageLayout(gtx, func(gtx C) D {
+				if sp.currentPage > 2 {
+					return layout.Flex{
+						Alignment: layout.Middle,
+						Axis:      layout.Vertical,
+					}.Layout(gtx,
+						layout.Rigid(func(gtx C) D {
+							lblTitle := sp.Theme.Label(values.TextSize16, sp.onBoardingScreens[sp.currentPage].title)
+							lblTitle.Font.Weight = font.Bold
+							return layout.Inset{Bottom: values.MarginPadding80}.Layout(gtx, lblTitle.Layout)
+						}),
+						layout.Rigid(func(gtx C) D {
+							return layout.Inset{Bottom: values.MarginPadding20}.Layout(gtx, sp.languageLayout)
+						}),
+						layout.Rigid(sp.setupButton),
+						layout.Rigid(func(gtx C) D {
+							gtx.Constraints.Min.X = gtx.Dp(values.MarginPadding570)
+							return layout.Inset{Top: values.MarginPadding20}.Layout(gtx, sp.nextButton.Layout)
+						}),
+					)
+				}
+				return layout.Flex{
+					Alignment: layout.Middle,
+					Axis:      layout.Vertical,
+				}.Layout(gtx,
+					layout.Rigid(func(gtx C) D {
+						list := &layout.List{Axis: layout.Horizontal}
+						return list.Layout(gtx, len(sp.onBoardingScreens), func(gtx C, i int) D {
+							if i == sp.currentPage {
+								return sp.pageSections(gtx, sp.onBoardingScreens[i])
+							}
+							return D{}
+						})
+					}),
+					layout.Rigid(func(gtx C) D {
+						return layout.Inset{
+							Top:    values.MarginPadding35,
+							Bottom: values.MarginPadding35,
+						}.Layout(gtx, sp.currentPageIndicatorLayout)
+					}),
+					layout.Rigid(func(gtx C) D {
+						gtx.Constraints.Min.X = gtx.Dp(values.MarginPadding350)
+						return sp.nextButton.Layout(gtx)
+					}),
+					layout.Rigid(func(gtx C) D {
+						return layout.Inset{Top: values.MarginPadding10}.Layout(gtx, func(gtx C) D {
+							gtx.Constraints.Min.X = gtx.Dp(values.MarginPadding350)
+							return sp.skipButton.Layout(gtx)
+						})
+					}),
+				)
+			})
+		}),
+	)
+
+}
+
+func (sp *startPage) languageLayout(gtx C) D {
+	// return layout.Flex{Axis: layout.Horizontal, Alignment: layout.Middle}.Layout(gtx,
+	return cryptomaterial.LinearLayout{
+		Width:       cryptomaterial.WrapContent,
+		Height:      cryptomaterial.WrapContent,
+		Orientation: layout.Horizontal,
+		Direction:   layout.Center,
+		Alignment:   layout.Middle,
+	}.Layout(gtx,
+		layout.Rigid(func(gtx C) D {
+			langTitle := sp.Theme.Label(values.TextSize16, values.String(values.StrLanguage))
+			langTitle.Font.Weight = font.Bold
+			return layout.Inset{Top: values.MarginPadding5}.Layout(gtx, langTitle.Layout)
+		}),
+		layout.Rigid(func(gtx C) D {
+			return layout.Inset{Top: values.MarginPadding5}.Layout(gtx, func(gtx C) D {
+				return layout.Stack{}.Layout(gtx,
+					layout.Expanded(func(gtx C) D {
```

Comment:
ANy reason why this is using Stack rather than just returning `return sp.onBoardingScreens[sp.currentPage].languageDropdown.Layout(gtx, 10, true)`

Reply:
Not responded to.